index.js

```diff
@@ -262,6 +271,10 @@ io.on('connection', function(socket){
                             }
                         }
 
+                        if (cell.build === builds.ROCK){ // Musíme odpočítať 10 energie
+                            energyCost = 10
+                        }
+
```

Comment:
Zbytečná duplicita.

Reply:
Teraz som si to uvedomil, bude upravené.